src/create.cpp

```diff
@@ -149,6 +151,9 @@ namespace create {
       }
     } // if CREATE_2
 
+    totalLeftDist += leftWheelDist;
+    totalRightDist += rightWheelDist;
```

Comment:
It may not be a serious issue, but there is potential for a wraparound here if someone is running the robot for the long term resulting in a negative distance.